### PR TITLE
Update _redirects

### PR DESCRIPTION
**Pull request by rbellary-vi**
@TheConnMan changing virtana URL for testing

### CODE REVIEW

**Review by rbellary-vi — COMMENTED** [review]
updating the URL

**Review by rbellary-vi — COMMENTED** [review]
c

**Review by rbellary-vi — COMMENTED** [review]
updated

**Review by rbellary-vi — COMMENTED** [review]
updareed

**Review by rbellary-vi — COMMENTED** [review]
updated

**Review by rbellary-vi — COMMENTED** [review]
updated

File: static/_redirects [hunk 1]
```diff
@@ -1,8 +1 @@
-/right-sizing/ec2-recommendations   /right-sizing/reports-ec2-recommendations/
-/right-sizing/resource-utilization   /right-sizing/reports-resource-utilization/
-/right-sizing/reports-ec2-reservations   /purchase-planning/reports-ec2-reservations/
-/right-sizing/resource-utilization   /right-sizing/reports-ec2-recommendations/
-/capacity-monitoring/metrics-page/   /capacity-monitoring/metrics/
-/integrations/linux-agent   /integrations/agents/linux-agent/
-/data-visualizaiton/metrics   /capacity-monitoring/metrics/
-/alerts-notificaitons/policies   /capacity-monitoring/policies/
+https://docs.metricly.com/* https://docs.virtana.com//:splat 301
```
Comment (TheConnMan) [inline static/_redirects:1]: Will this redirect each page to the same type of page in docs.virtana.com or will this redirect to the same exact URI over on docs.virtana.com? For example, https://docs.metricly.com/idle-resources/ needs to redirect to https://docs.virtana.com/en/idle-resources.html.

Comment (rbellary-vi) [reply static/_redirects:1]: @TheConnMan redirection is not working yet working on that will update once it was done

Comment (TheConnMan) [reply static/_redirects:1]: Ok, sounds good. Please re-request a review from me when you're ready for review.

Comment (rbellary-vi) [reply static/_redirects:1]: @TheConnMan sure , But sorry about this many edits

**Review by rbellary-vi — COMMENTED** [review]
updated

**Review by rbellary-vi — COMMENTED** [review]
up

**Review by rbellary-vi — COMMENTED** [review]
u

**Review by rbellary-vi — COMMENTED** [review]
u

**Review by rbellary-vi — COMMENTED** [review]
up

**Review by jpadavala-vi — APPROVED** [review]
test

**Pull request closed without merging**